.gitignore

```diff
@@ -9,3 +9,6 @@ settings.py
 *.pyc
 .idea/*
 .cache/*
+docs/_build/
+docs/sanic.rst
+docs/modules.rst
```

Comment:
I think it would be less confusing if the apidoc files are generated in a _* folder